pttrack/templates/pttrack/document_detail.html

```diff
@@ -28,10 +28,15 @@
     <small class="text-muted">uploaded by {{ document.author }} on {{ document.written_datetime }}</small>
     {% if document.last_modified != document.written_datetime %}
     <br><small class="text-muted">last modified {{document.last_modified}}</small>
-    {% endif %}</p>
+    {% endif %}
+  <div class="row">
+    <h3><br>&nbsp;&nbsp;<a href="{{ document.image.url }}">Link to File</a></h3>
+    <small class="text-muted">&nbsp;&nbsp;&nbsp;&nbsp;(Preview and Comments Below)</small>
+  </div>
   <div class="row ">
-    <img class="img-rounded img-responsive center-block" src="{{ document.image.url }}" alt="alt_text">
+    <img class="img-rounded img-responsive center-block" src="{{ document.image.url }}" alt="Please Click Link Below">
```

Comment:
technically speaking, we should be using the alt-text to describe the file, and not to give instructions (it's an accessibility feature for blind people, I think). That being said, I'm not sure how important that is.